pkg/onchain/website/uploader.go

```diff
@@ -112,6 +113,60 @@ func upload(client *node.Client, addr []byte, chunks []string, wallet *wallet.Wa
 	return &operations, nil
 }
 
+//nolint:lll
+func UploadMissedChunks(atAddress string, content string, wallet *wallet.Wallet, missedChunks string) (*[]string, error) {
```

Comment:
I have a concern about the naming of thoose both functions ( they are the same, delta the CAP letter ) 

Also I think that you can return the value array and not it's pointer.
@gregLibert can you confirm ?

Reply:
Naming function is ok

Saw with @SebastienValla on vocal